src/modules/i18n/components/I18nProvider.js

```diff
@@ -26,26 +26,13 @@ type StateType = {|
 |}
 
 class I18nProvider extends React.Component<PropsType, StateType> {
-  i18n: i18n
```

Comment:
Would be nice to have this variable instead of working on the global i18n object